frontend/src/routes/CumulativeMonthlyWorkingHours.jsx

```diff
@@ -34,7 +35,26 @@ const CumulativeMonthlyWorkingHours = () => {
           transformedTableData[item.year][item.month] = cumulativeTableHours;
           transformedTableData[item.year].total += item.total_hours;
         }
-        setTableData(Object.values(transformedTableData));
+        const objectValues = Object.values(transformedTableData);
+
+        const roundToInt = (num) => {
```

Comment:
Do you need an additional function "roundToInt"? I think using Math.round directly would be simpler.